### ISSUE TITLE
I can see a single nextdoor nature resource

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/teamwilder#37]
## Acceptance Criteria in order of priority and should be converted to own issues

- [ ] A nextdoor nature resource has a title
- [ ] A nextdoor nature resource has a text description with headings
- [ ] A nextdoor nature resource can contain a video guide
- [ ] A nextdoor nature resource can contain associated case studies
- [ ] A nextdoor nature resource can contain associated guides
- [ ] #55 TBC
- [ ] #57

## Implementation plan

- [x] Create data type for resource
- [x] Pipe resource data to view using slug as id (NOTE will do actual lookup after #48 )
- [x] Remove resource index route
- [x] Add dummy resource
- [x] Create teaser data type for case study & resource
- [x] Sweep for [fff] and recard as issues to do OR complete
- [x] add tests
- [ ] #56

**Comment by ivan-kocienski-gfsc** [comment]
Is there some design on this? Or notes?

**Comment by katjam** [comment]
Let me know if you need access to the Miro. The design is more of a mock-up at this stage - so we're putting elements in rough order and with the content they belong with. More concerned with data here than actual design. Only some of the things in the image are in that list above. Other stuff is not a priority right now.

In terms of even audio and video, we need to check they actually have this content so I suspect those items should also be pushed to the bottom.

![image](https://user-images.githubusercontent.com/861172/224112665-da14db46-b34d-472f-8215-e48effedcbf8.png)

**Comment by r-ferrier** [comment]
verified:

![Screenshot 2023-05-29 at 16 23 12](https://github.com/geeksforsocialchange/teamwilder/assets/46496391/9ba82560-10ab-4431-946a-3703a10cedca)
